style.css

```diff
@@ -129,4 +137,171 @@ header {
     margin: 0 0;
     font-size: 2rem;
     font-weight: normal;
+}
+.recent-projects-section {
+    padding: 50px 20px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.recent-projects-parent {
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: center;
+    gap: 30px;
+    max-width: 800px;
```

Comment:
you have two elements with `max-width: 800px`. i assume that means that you want them both to be the same width. in that case, it would be good to put that common styling in one place, so that if we decide that it needs to be changed e.g. to 700px, it can be changed one time and apply to both at the same time, rather than changing one and forgetting about the other.